requirements.sh

```diff
@@ -17,19 +17,62 @@
 ##--------------------------------------------------------------------
 
 ##
-## Author: Ashish Jabble
+## Author: Ashish Jabble, Massimiliano Pinto
 ##
 
 
 set -e
 
-sudo apt update
-sudo apt -y upgrade
+foglamp_location=`pwd`
+is_rhel=`(lsb_release -ds 2>/dev/null || cat /etc/*release 2>/dev/null | head -n1 || uname -om) | egrep '(Red Hat|CentOS)' || echo ""`
+if [ "${is_rhel}" != "" ]; then
+	echo "Platform is ${is_rhel}"
+	sudo yum check-update
```

Comment:
This script runs only with sudo, so the "sudo" prefix for these yum commands are not required. Same for apt commands below also.